src/app/models/work-item-type.ts

```diff
@@ -106,11 +100,11 @@ export class WorkItemTypeMapper implements Mapper<WorkItemTypeService, WorkItemT
         toPath: ['type'],
         toValue: 'workitemtypes'
       }, {
-        fromPath: ['relationships', 'infotip', 'data'],
-        toPath: ['infotip'],
+        fromPath: ['attributes', 'description'],
+        toPath: ['description'],
         toFunction: function(value) {
```

Comment:
As we are not doing much here, let's compact the code, `(value) => value ? value : ''`  This will save 6 lines.